LidarrAPI/Startup.cs

```diff
@@ -81,7 +81,6 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory, IApplicationLifetime applicationLifetime)
         {
             loggerFactory.AddNLog();
```

Comment:
`UseNLog` replaces both `app.AddNLogWeb();` and `loggerFactory.AddNLog();`. Keeping `AddNLog()` will give issue with double logging because of two logging providers.

Reply:
Thanks for the catch! Should be resolved now.